andela/ah-orcas
# feat(follow profile): add follow and unfollow another user profile

- authenticated user should be able to follow another user
- authenticated user should be able to un-follow anothe user
- a user should be able to follow themselves
- create tests for following

[Delivers #161254668]

#### What does this PR do?
```Adds functionality for users to follow and unfollow other users ```
#### Description of Task to be completed?
```
an authenticated user hitting api/profiles/<username>/follow/
with POST request should be able to follow the user
with DELETE request should be able to unfollow the user
 ```
#### How should this be manually tested?
$ virtualenv env
$ source env/bin/activate
$ cd env
```
- Clone this repo.
``` 
$ git clone https://github.com/andela/ah-orcas.git
```
- Switch to this branch
``` 
$  git checkout  ft-users-follow_each_other_profile-161254668
```
- Create a .env file and run it. Check sample.env file to configure yours.
```
$ source .env
```
- Run the app
```
$ python manage.py runserver
#### Any background context you want to provide?
```Users cannot follow themselves```
#### What are the relevant pivotal tracker stories?

[#161254668](https://www.pivotaltracker.com/story/show/161254668)
